Source/Urho3D/Physics/RaycastVehicle.h

```diff
@@ -168,6 +168,8 @@ class URHO3D_API RaycastVehicle : public LogicComponent
     WeakPtr<RigidBody> hullBody_;
     /// Opaque Bullet data hidden from public
     RaycastVehicleData* vehicleData_;
+    /// Coordinate system
```

Comment:
Terminate the doxygen comment with a period. Also, may be declare an enumeration or constants for the coordinate system. They should be more readable than 0, 1, and 2.